Odata-docs/client/using-dataservice-ihttpclientfactory.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Configuring HttpClient with IHttpClientFactory in Microsoft.OData.Client"
+description: "This tutorial provides guidance on configuring HttpClient with IHttpClientFactory in OData.NET 8"
+
+author: WanjohiSammy
+ms.author: swanjohi
+ms.date: 11/29/2024
+ms.topic: article
+---
+
+# Configuring HttpClient with IHttpClientFactory in Microsoft.OData.Client
+
+This feature was introduced in [OData.NET 8](/odata/odatalib/release-notes/odatalib-8#added-support-for-ihttpclientfactory).
+
+`Microsoft.OData.Client` uses `HttpClient` by default, but this document will guide you on how to configure `HttpClient` by providing settings like timeouts and custom handlers using `IHttpClientFactory`.
```

Comment:
Since this is the first time you're mentioning `HttpClient`, consider making it a link
```suggestion
`Microsoft.OData.Client` uses [`HttpClient`](https://learn.microsoft.com/dotnet/api/system.net.http.httpclient) by default, but this document will guide you on how to configure `HttpClient` by providing settings like timeouts and custom handlers using `IHttpClientFactory`.
```